.storybook/themes/news.scss

```diff
@@ -685,6 +685,39 @@
 				right-rail-advanced-main-right-rail-bottom: (
 					grid-area: main-right-rail-bottom,
 				),
+				share-bar: (
+					background: var(--background-color),
+					box-shadow: var(--global-box-shadow-1),
+					border-radius: 0 var(--border-radius) var(--border-radius) 0,
+					display: none,
```

Comment:
this might be a dumb idea: but if there's display none for default, would the `desktop` breakpoint have all of the styles anyway? Would this reduce css duplication?

Reply:
You are not being dumb - as it's mobile first the styles are applied here and we just don't display the block - the desktop toggles the display value